### PR TITLE
Add remote platform docs for Kaliedescape integration

### PR DESCRIPTION
**Pull request by SteveEasley**
## Proposed change

Add remote platform documentation for new Kaleidescape integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/67959
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/kaleidescape.markdown [hunk 1]
```diff
@@ -35,6 +37,33 @@ This integration is intended for the automation of Kaleidescape players with a m
 
 The Kaleidescape media player platform will create a Media Player entity for the device. This entity will display the currently playing media and playback controls.
 
+## Remote
+
+The Kaleidescape remote platform will create a Remote entity for the device. This entity allows you to send the following commands via the `remote/send_command` service.
```
Comment (bdraco) [inline source/_integrations/kaleidescape.markdown]:
```suggestion
The Kaleidescape remote platform will create a Remote entity for the device. This entity allows you to send the following commands via the `remote.send_command` service.
```

File: source/_integrations/kaleidescape.markdown [hunk 2]
```diff
@@ -35,6 +37,33 @@ This integration is intended for the automation of Kaleidescape players with a m
...
+- left
+- right
+- cancel
+- replay
+- scan_forward
+- scan_reverse
+- go_movie_covers
+- menu_toggle
+
+A typical service call might look like the example below. This sends a command to the device to `select` the currently highlighted item.
```
Comment (bdraco) [inline source/_integrations/kaleidescape.markdown]:
```suggestion
A typical service call might look like the example below, which sends a command to the device to `select` the currently highlighted item.
```

File: source/_integrations/kaleidescape.markdown (hunk 1 above)
Comment (SteveEasley) [reply source/_integrations/kaleidescape.markdown]: Fixed.

File: source/_integrations/kaleidescape.markdown (hunk 2 above)
Comment (SteveEasley) [reply source/_integrations/kaleidescape.markdown]: Fixed.

**Review by bdraco — APPROVED** [review]

**Comment by SteveEasley** [comment]
@bdraco Should we be worried about the missing logo on https://next.home-assistant.io/integrations/kaleidescape? Not sure how brands work. The specific url used in that page isnt working.

Works: https://brands.home-assistant.io/kaleidescape/logo@2x.png
Doesn't: https://brands.home-assistant.io/_/kaleidescape/logo@2x.png

**Comment by bdraco** [comment]
both are loading for me.  It is likely a cache issue.  We don't clear it until right before release

File: source/_integrations/kaleidescape.markdown [hunk 3]
```diff
@@ -35,6 +37,33 @@ This integration is intended for the automation of Kaleidescape players with a m
...
+- up
+- down
+- left
+- right
+- cancel
+- replay
+- scan_forward
+- scan_reverse
+- go_movie_covers
+- menu_toggle
```
Comment (frenck) [inline source/_integrations/kaleidescape.markdown:54]: These are code/commands, which should have been code-fenced.

Please open up a PR to fix that.

File: source/_integrations/kaleidescape.markdown [hunk 4]
```diff
@@ -35,6 +37,33 @@ This integration is intended for the automation of Kaleidescape players with a m
 
 The Kaleidescape media player platform will create a Media Player entity for the device. This entity will display the currently playing media and playback controls.
 
+## Remote
+
+The Kaleidescape remote platform will create a Remote entity for the device. This entity allows you to send the following commands via the `remote.send_command` service.
```
Comment (frenck) [inline source/_integrations/kaleidescape.markdown:42]: Let's link the `remote.send_command` service to the appropriate documentation,

File: source/_integrations/kaleidescape.markdown (hunk 3 above)
Comment (SteveEasley) [reply source/_integrations/kaleidescape.markdown:54]: Fixed in https://github.com/home-assistant/home-assistant.io/pull/22006.

File: source/_integrations/kaleidescape.markdown (hunk 4 above)
Comment (SteveEasley) [reply source/_integrations/kaleidescape.markdown:42]: Fixed in https://github.com/home-assistant/home-assistant.io/pull/22006.

**Pull request merged**